pizza.html

```diff
@@ -18,41 +19,79 @@
             </ul>
         </nav>
     </header>
+    
     <!-- Content of the Website -->
     <h1>Pizza</h1>
-    <div class="infobox">
-        <img src="#" alt="piccante" />
-        Picante 16$<br>
-        <small>Tomato, mozzarella, spicy salami, chillies, oregano</small>
+    <div id="piccante" class="infobox">
+        <img src="#" alt="Piccante" />
+        <h2>Piccante</h2>
+		<p class="indegrients">Tomato, mozzarela, spicy salami, chillies, oregano</p>
+		<p class="preis">16$</p>
```

Comment:
In Englisch heisst's: "ingredients"
Und ich würde einheitlich entweder nur auf Englisch oder nur auf Deutsch die class-Namen verteilen. Also hier auch "price" anstatt "preis"

Reply:
Die verschiedenen divs werden später unterschiedlich positioniert und müssen daher eindeutig identifizierbar sein.